<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLogin</class>
 <widget class="QDialog" name="UserLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
font: 18pt &quot;微软雅黑&quot;;
border: 0px solid rgb(0, 125, 126);
border-radius:8px;
margin-top:10px;
}
QGroupBox:title{
color:white;
subcontrol-origin: margin;
subcontrol-position: top center;
left: 5px;
}
QToolButton{
font: 20pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(0, 125, 165);
}
QLineEdit{
font: 24pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid rgb(0, 125, 165);
border-radius:10px;
}
QLabel{
font: 20pt &quot;微软雅黑&quot;;
color:rgb(15, 41, 68);
}
QDialog{
background-color: rgb(255, 255, 255);
border:2px solid rgb(0, 125, 165);
border-radius:10px;
}
</string>
  </property>
  <widget class="QToolButton" name="tBtnAdmin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>270</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLogin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>235</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="icon">
    <iconset resource="../image.qrc">
     <normaloff>:/Image/ok.png</normaloff>:/Image/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnHide">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>235</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="icon">
    <iconset resource="../image.qrc">
     <normaloff>:/Image/close.png</normaloff>:/Image/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbUser">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/pic/login.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/pwd.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPwd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>270</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string> 请输入密码......</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>384</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入用户登录密码!!!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>85</y>
     <width>270</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(0, 125, 165);</string>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>265</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>V1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
